client/site-profiler/components/basic-metrics/styles.scss

```diff
@@ -1,22 +1,86 @@
+@import "@automattic/typography/styles/variables";
+@import "@wordpress/base-styles/breakpoints";
+
 .basic-metric-details {
-	& .good {
-		color: #019c3b;
+	display: grid;
+	grid-template-columns: repeat(3, 1fr);
+	grid-template-rows: repeat(2, 1fr);
+	grid-column-gap: 72px;
+	grid-row-gap: 72px;
 
+	@media (max-width: $break-large ) {
+		grid-template-columns: 1fr;
 	}
-	& .needs-improvement {
-		color: #d4bb4c;
+}
+
+
+.basic-metrics__card {
+	.basic-metrics__header {
+		padding-left: 28px;
+		padding-top: 12px;
+		padding-bottom: 8px;
+		margin-bottom: 24px;
+		border-left: 5px solid;
+
+		.basic-metrics__name {
+			color: var(--studio-gray-20);
+			display: flex;
+			gap: 4px;
+		}
+
+		.basic-metrics__value {
+			display: inline-block;
+			background-clip: text;
+			color: transparent;
+			-webkit-background-clip: text;
+			-webkit-text-fill-color: transparent;
+			background-color: var(--studio-gray-30);
+		}
+
+		&.good {
+			border-color: #359f4c;
```

Comment:
It might be a good idea if we can use any of the pre-defined Calypso colors.

Reply:
Added a PR for these: https://github.com/Automattic/wp-calypso/pull/91062